mifosng-android/src/main/java/com/mifos/mifosxdroid/activity/login/LoginActivity.kt

```diff
@@ -37,6 +37,10 @@ class LoginActivity : MifosBaseActivity() {
                         supportFragmentManager,
                         "UpdateServerConfigFragment"
                     )
+                },
+                onSuccessNavigate = {
+                    startActivity(Intent(this, HomeActivity::class.java))
+                    finish()
```

Comment:
why are we adding this? In compose we only need one activity, this `LoginActivity.kt` file should be removed